src/messages.rs

```diff
@@ -302,9 +302,9 @@ impl SignedMessage {
         utils::decode::<RoutingMessage>(&self.encoded_body)
     }
 
-    pub fn encoded_body(&self) -> &Vec<u8> {
```

Comment:
these functions have been marked with #[allow(dead_code)]; sorry for the conflict